### DISCUSSION TITLE
现在13.13版本了为什么装备还是13.12的装备

### DISCUSSION THREAD

**Discussion by Kingbirdzmr** [cangzhang/ChampR#283] [General]
甚至有一种感觉就是按了apply builds新的出装没有导入

**Comment by cangzhang** [comment]
https://github.com/cangzhang/champ-r/actions/runs/5431793260

来试试这里的版本，会显示导入的状态。
装备更新需要周期，但是我看数据源都是更新了的，所以你重新定期导入下

**Reply by Kingbirdzmr** [reply]
我试一下，我是每次开游戏的时候都会重新导入出装

**Reply by cangzhang** [reply]
数据源每天凌晨更新一次，所以你每天导入一次就够了

**Reply by Kingbirdzmr** [reply]
![1688475881548](https://github.com/cangzhang/champ-r/assets/43699171/3ccddd76-b541-4b81-838f-142008d95b47)
刚刚下载了那个版本导入了一下还是只有13.12的数据，那个13.09的数据是1.x版本没有删掉的

**Reply by cangzhang** [reply]
你的 LoL 安装路径是什么？
现在的 ChampR 每次导入会默认清除旧出装

**Reply by Kingbirdzmr** [reply]
D:\Games\LOL\LOL\Riot Games\League of Legends
刚刚下回了1.3版本的champr可以导入
![1688476357713](https://github.com/cangzhang/champ-r/assets/43699171/c69495a5-da29-4932-99a0-c1f3b4daf18b)

**Reply by cangzhang** [reply]
什么服务器？看起来不是马服

**Reply by Kingbirdzmr** [reply]
台服

**Reply by cangzhang** [reply]
可以先用之前的版本（v2.0.1-fix 及更早的版本），后面会适配马服之外的服务器

**Comment by cangzhang** [comment]
台服可以试试这里的版本 https://github.com/cangzhang/champ-r/actions/runs/5566797314
